app/src/main/java/com/twilio/video/app/ui/room/RoomActivity.java

```diff
@@ -500,28 +509,10 @@ void connectButtonClick() {
         if (text != null) {
             final String roomName = text.toString();
 
-            roomViewModel.connectToRoom(
-                    displayName,
-                    roomName,
-                    getLocalAudioTracks(),
-                    getLocalVideoTracks(),
-                    isNetworkQualityEnabled());
+            roomViewModel.connectToRoom(displayName, roomName, isNetworkQualityEnabled());
```

Comment:
Does this introduce a bug where the user is capturing their screen and then tries to join a room?

Reply:
Is there a good reason to allow the user to share their screen before entering a room? We should only allow a user to share their screen once they have joined a room right?

Reply:
I suppose we don't have to allow this, but the screen share UI needs to be updated to accommodate this in order for the PR to be merged. Namely, the screen share button needs to be disabled before reaching the connected state.

Reply:
Yes I agree with that. I didn't realize that the screen share button was available before joining a room 😆. I'll get that change pushed and have you review before merging.